pkg/controller/fileshare/filesharecontroller_test.go

```diff
@@ -226,3 +226,11 @@ func (fc *fakefileshareClient) AttachVolume(ctx context.Context, in *pb.AttachVo
 func (fc *fakefileshareClient) DetachVolume(ctx context.Context, in *pb.DetachVolumeOpts, opts ...grpc.CallOption) (*pb.GenericResponse, error) {
 	return nil, nil
 }
+
+func (fc *fakefileshareClient) GetMetrics(ctx context.Context, in *pb.GetMetricsOpts, opts ...grpc.CallOption) (*pb.GenericResponse, error) {
+	return nil, nil
+}
+
+func (fc *fakefileshareClient) GetUrls(ctx context.Context, in *pb.NoParams, opts ...grpc.CallOption) (*pb.GenericResponse, error) {
```

Comment:
What are these two added?  They are not specific to Thin OpenSDS.

Reply:
I know, but to enable the interface implement function, we have to add these two methods.

Reply:
Here is an example: https://github.com/opensds/opensds/blob/thin-opensds/pkg/api/controllers/volume.go#L44

Reply:
I see. I think these changes should be submitted to Development branch as a separate bug fix in Capri.

Reply:
Got it, I will do that